### ISSUE TITLE
reservation buffer necessary?

### ISSUE THREAD

**Issue by birnam** [luciodj/Rocket#1]
I'm pretty green in both PIC and C++, so this might be obvious and I just haven't figured it out.

Is line 24 in #3HEF main.c necessary? 

`char    buffer[ HEF_BLOCKSIZE];`

It's unused in the code, so the only thing I've been able to come up with is that it serves to reserve a chunk of memory that won't be used by the compiler. However this is what the `--ROM` option does. Which, if I'm correct so far, makes me think that `buffer` might be leftover code from an earlier iteration that didn't use `--ROM`.

**Comment by luciodj** [comment]
You are correct, that is dead code left in main.c after I refactored the example.
It does not serve any purpose and can be removed!
